src/main/java/edu/kis/powp/jobs2d/drivers/adapter/FiguresJaneAdapter.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJane;
+import edu.kis.powp.jobs2d.AbstractDriver;
+
+public class FiguresJaneAdapter extends AbstractDriver {
+    private Job2dDriver driver;
+
+    public FiguresJaneAdapter(Job2dDriver driver) {
+        super(0, 0); // Początkowa pozycja (0,0)
+        this.driver = driver;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        driver.operateTo(x, y);
```

Comment:
AbstractDriver already has some implementation, which should be taken into an account.
Direct SetPosition calls are ignored so this Adapter would not work well.